CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/grid/GridArranger.java

```diff
@@ -276,37 +274,39 @@ public int getTopLeftVxId() {
     }
 
     /**
-     * Given the choice of grid (one of the GRID_CHOICE_xxx constants), return a
-     * setting from number of rows and number of columns. Zero means
-     * unconstrained.
+     * Given the choice of grid (one of the GRID_CHOICE_xxx constants), return a setting from number of rows and number
+     * of columns. Zero means unconstrained.
      *
      * @param gc The kind of grid to do the arrangement with.
      *
      * @return A Dimension corresponding to the choice of grid.
      */
     private static Dimension getGridSize(final GridChoice gc) {
-        if (gc == null) {
+        if (null == gc) {
             return new Dimension(0, 0);
-        } else if (gc == GridChoice.SQUARE) {
-            return new Dimension(0, 0);
-        } else if (gc == GridChoice.HORIZONTAL_LINE) {
-            return new Dimension(0, 1);
-        } else if (gc == GridChoice.VERTICAL_LINE) {
-            return new Dimension(1, 0);
-        } else if (gc == GridChoice.TWO_ROWS) {
-            return new Dimension(0, 2);
-        } else if (gc == GridChoice.THREE_ROWS) {
-            return new Dimension(0, 3);
-        } else if (gc == GridChoice.FOUR_ROWS) {
-            return new Dimension(0, 4);
-        } else if (gc == GridChoice.TWO_COLUMNS) {
-            return new Dimension(2, 0);
-        } else if (gc == GridChoice.THREE_COLUMNS) {
-            return new Dimension(3, 0);
-        } else if (gc == GridChoice.FOUR_COLUMNS) {
-            return new Dimension(4, 0);
+        } else {
+            switch (gc) {
+                case SQUARE:
+                    return new Dimension(0, 0);
```

Comment:
Since this code is the same as the default case, you can probably omit this case